plugins/woocommerce/includes/admin/class-wc-admin-post-types.php

```diff
@@ -200,7 +201,7 @@ public function order_updated_messages( array $messages ) {
 			9  => sprintf(
 			/* translators: %s: date */
 				__( 'Order scheduled for: %s.', 'woocommerce' ),
-				'<strong>' . date_i18n( __( 'M j, Y @ G:i', 'woocommerce' ), strtotime( $theorder->get_date_created() ) ) . '</strong>'
+				'<strong>' . date_i18n( __( 'M j, Y @ G:i', 'woocommerce' ), ($theorderDateCreated ? strtotime( $theorderDateCreated ) : false) ) . '</strong>'
```

Comment:
At its core, the issue here is that `$theorder->get_date_created()` returns `NULL` in this context (as the order hasn't been saved yet). I believe we could do this differently and more succinctly (and skip the assignment above):

```suggestion
				'<strong>' . date_i18n( __( 'M j, Y @ G:i', 'woocommerce' ), ( $theorder->get_date_created() ?? $post->post_date ) . '</strong>'
```

Reply:
I changed it to `$theorder->get_id() !== 0 ? strtotime( $theorder->get_date_created() ) : false`, is it ok now ?

Reply:
It's not because `$theorder` should always have an ID. What it doesn't necessarily have is a created date, but in those cases we could use the `$post->post_date` which is what my suggestion did. Alternatively, we could also try to use the current time, but if `$post->post_date` works, I'd vote for that instead.

I also added this bit of feedback to my latest review. Here to help if you have any other questions!